### PR TITLE
docs: fix theme name, doc references and add PR template

### PR DESCRIPTION
**Pull request by arslanashraf7**
## Related Ticket:
None

## What this PR does?
- Fixes the theme name
- Fixes commands (paver-update)
- Adds PR template

## How to test?
- Just a general look is good, no need to build or test the theme itself
- Make sure the reference links now point to the right URLs and don't throw 404

### CODE REVIEW

**Comment by asadiqbal08** [comment]
Change set LGTM 👍  but just a question, why I am seeing xpro references in this guide ?
https://github.com/mitodl/mitxonline/blob/main/docs/source/configuration/open_edx.rst

Is the guide not related to mitxonline ?

**Review by asadiqbal08 — APPROVED** [review]
👍 Else seems good but a question about another the guide that is pinned in this.

**Comment by arslanashraf7** [comment]
> Change set LGTM 👍 but just a question, why I am seeing xpro references in this guide ? https://github.com/mitodl/mitxonline/blob/main/docs/source/configuration/open_edx.rst
> 
> Is the guide not related to mitxonline ?

@asadiqbal08 The doc reference is related to the MITxOnline and it points to the readme of MITxOnline.
But the IDP-related/Auth related extensions are common for both xPRO and MITxOnline e.g. `social-auth-mitxpro  and mitxpro-openedx-extensions` so they will have to use the xpro naming conventions.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#### Pre-Flight checklist
+
+- [ ] Screenshots and design review for any changes that affect layout or styling
+  - [ ] Desktop screenshots
+  - [ ] Mobile width screenshots
+- [ ] Migrations
+  - [ ] Migration is backwards-compatible with current production code
```
Comment (pdpinch) [inline .github/pull_request_template.md:7]: I don't think this is relevant to this repo?

**Review by pdpinch — APPROVED** [review]
This is fine as is, but I left a couple of comments.

File: .github/pull_request_template.md (hunk 1 above)
Comment (arslanashraf7) [reply .github/pull_request_template.md:7]: I agree we won't be having migrations in this repo generally.

**Pull request merged**